app/products/calls/screens/call/call_screen.tsx

```diff
@@ -297,6 +308,14 @@ const CallScreen = (props: Props) => {
         props.actions.setSpeakerphoneOn(!props.speakerphoneOn);
     };
 
+    const toggleRaiseHand = () => {
```

Comment:
useCallback ?

Reply:
Yah, I just think we're probably overdoing the useCallbacks. From my understanding, the recommendation is to not use it unless profiling shows you're doing too many unnecessary rerenders -- and here we're making useCallbacks for simple functions that are used in single buttons, it's probably adding more overhead than you're saving (and more garbage to be collected). But since I don't have the profiling evidence, I don't know either way. I guess I would be more on the side of "default to the simple way, then add useCallback if we can prove it's worth it."  Anyway, it's something we as a team can talk about for the future. 
(Some good reading, which you've probably already seen, but just in case you haven't: https://kentcdodds.com/blog/usememo-and-usecallback )

Anyway, I added the useCallbacks to make it consistent with how it's been done in the rest of the component.